packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -377,6 +377,23 @@ module.exports.subscriptionsSubscriptionSupportValidator = isA.object({
   status: isA.string().required(),
   subscription_id: module.exports.subscriptionsSubscriptionId.required(),
 });
+module.exports.subscriptionsPlaySubscriptionSupportValidator = isA.object({
+  auto_renewing: isA.bool().required(),
+  cancel_reason: isA.number().optional(),
+  expiry_time_millis: isA.number().required(),
+  package_name: isA.string().optional(),
+  sku: isA.string().optional(),
+  product_id: isA.string().optional(),
```

Comment:
Are these optional, because they aren't currently used by the support panel?

Reply:
Yes.